README.md

```diff
@@ -11,6 +11,20 @@ The MSAL library for Android gives your app the ability to use the [Microsoft Cl
 
 [![Version Badge](https://img.shields.io/maven-central/v/com.microsoft.identity.client/msal.svg)](https://repo1.maven.org/maven2/com/microsoft/identity/client/msal/)
 
+## AI Development Resources
+
+For AI development tools like Cline that access this repository through the GitHub web interface, these resources provide comprehensive guidance for MSAL integration without requiring a local repository clone:
+
+- [AI Assistant Guidelines](Ai.md) - Primary documentation for AI tools, containing core principles and patterns
+- [Cline Rules](.clinerules/msal-cline-rules.md) - Essential best practices that Cline must follow for generating MSAL code
+- [Example Applications](examples/) - Complete production-quality implementations:
+  - [Multiple Account Example](examples/hello-msal-multiple-account/) - Shows multiple account authentication pattern
+  - [Single Account Example](examples/hello-msal-single-account/) - Shows single account authentication pattern
+- [Code Snippets](snippets/) - Ready-to-use code samples for common MSAL operations
+- [Configuration Template](auth_config.template.json) - Reference template showing all required and optional settings
+
+These resources contain all necessary patterns, rules, and examples for AI tools to assist with MSAL integration, providing proper API usage, configuration, and implementation patterns for both multiple and single account scenarios.
+
 ### :exclamation: Migrating from ADAL
```

Comment:
It seems this has now moved as a sub-heading under `AI Development Resources`. Let's fix it.